developer/src/common/web/utils/src/keyman-urls.ts

```diff
@@ -0,0 +1,24 @@
+export class KeymanUrls {
+  static readonly HELP_KEYMAN_COM = `https://help.keyman.com`;
+  static readonly KEYMAN_COM = `https://keyman.com`;
+
+  // Various Sites
+  static readonly NEW_KEYMAN_ISSUE = ()       => 'https://github.com/keymanapp/keyman/issues/new';
+  static readonly LDML_SPEC = (topic: string) => `https://www.unicode.org/reports/tr35/tr35-keyboards.html#${topic}`;
+
+  // TODO: Not formatting code here to avoid dep on @keymanapp/common-types for developer-utils
+  static readonly COMPILER_ERROR_CODE = (code: string) => `https://kmn.sh/${code}`; // code should be km##### (already formatted)
+
+  // help.keyman.com
+  static readonly VIRTUAL_KEYS = ()         => `${KeymanUrls.HELP_KEYMAN_COM}/developer/language/guide/virtual-keys#common-virtual-key-codes`;
+  static readonly FILE_TYPE = (ext: string) => `${KeymanUrls.HELP_KEYMAN_COM}/developer/current-version/reference/file-types/${ext}`;
+  static readonly KMN_REF = (topic: string) => `${KeymanUrls.HELP_KEYMAN_COM}/developer/language/reference/${topic}`;
+  static readonly HELP_KEYBOARD = (id: string)   => `${KeymanUrls.HELP_KEYMAN_COM}/keyboard/${id}`;
+  static readonly HELP_MODEL = (id: string)      => `${KeymanUrls.HELP_KEYMAN_COM}/model/${id}`;
```

Comment:
Is /model a TODO?

Reply:
I don't understand the question?

Reply:
I thought help documentation for models aren't on help.keyman.com yet.

Reply:
Gotcha. The link was already there in .model_info files, so this doesn't change anything from Developer side. The TODO remains somewhere, doesn't need to be tracked here.